src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/EETypeNode.cs

```diff
@@ -565,7 +565,6 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly)
             ComputeOptionalEETypeFields(factory, relocsOnly);
 
             OutputGCDesc(ref objData);
-            OutputComponentSize(ref objData);
```

Comment:
Could you update the large comment at the top of the file? We no longer emit/access this as two ushorts, but as a single int.

Reply:
Updated